### ISSUE TITLE
Hardware Issue with Device - LCD Lifting off PCB

### ISSUE THREAD

**Issue by jwshields** [jamesbowman/i2cdriver#56]
Hello,  
I'm pretty sure this is not the right place to open a hardware issue for this device, but I've sent an email to the `jamesb at excamera dot com` address and I've not received a response..  
I've been experiencing an issue with my I2CDriver where the board flexes a bit much (Maybe it is just me being a brute?) - but if I'm fiddling with it and plugging jumpers in, or the plugging/unplugging the USB cable; when I am doing those things, I seem to see/feel the board flexing in ways which are exerting forces upon the LCD in an unnatural way.  (I also own an SPIDriver and suspect this issue could also happen with that device as well, but I am not experiencing it with my device currently)
  
A few nights ago when I was using the I2C driver, I noticed that the LCD had lifted off the PCB, because light from the backlight module was leaking out of the sides.  

I’ve taken some photos of it, and outlined in red boxes what I’m seeing; in the photos where you see my fingers applying some pressure, I’m doing that to attempt to demonstrate how the LCD is loose and how the leakage of the backlight changes.  
The photos are available here: https://file.nouille.party/i2c_driver/  

... Onto my actual problem/questions!
- Is this a phenomenon that has been seen before with either the I2C Driver or SPI Driver?
- Are there any recommendations on how to secure the LCD, such that I can do it myself, or what the best thing to do here may be?

Mind, the functionality of the device is not compromised. I just don’t want my LCDs falling off and cracking, or becoming any more loose than what I’m seeing on my I2C Driver. And I am only mentioning the SPI Driver because I feel like it could be possible this issue exists on that board as well.  

Thank you in advance, and, I apologize for opening an issue here, as I don't know if this repo is an appropriate place for hardware issues. If so, please feel free to close this issue.

**Comment by jwshields** [comment]
I'll try some glue, either thinking I'll try some form of super glue that I can gently dab onto the board. Will post results once I do that.
But... I do hope Mr. Bowman sees this issue and can provide some insight and/or help

**Comment by GOTO-GOSUB** [comment]
In the end I used a latex based glue (Copydex) and just ran a little beading of it around the top of the display and a dab in each of the bottom corners then applied just enough pressure to hold the display against the PCB while it dried. So far it's holding up well.

**Comment by jwshields** [comment]
Necro'ing this issue a bit. Life has kept me a bit busy..  
A few updates:
1. I heard back from James, apparently I completely missed the support email for i2cdriver on the website, but he gave more or less the same pointers as you did @GOTO-GOSUB 
2. I tested on both my I2CDriver & SPIDriver boards. I put superglue around the plastic casing that held the LCD on, and superglue between the PCB and the plastic casing. It worked amazingly to secure the LCDs.
3. In my excitement to test things out, I accidentally shorted out gnd and vcc and borked my I2CDriver!

Closing this issue, as, it is technically resolved for me, and the superglue method worked wonderfully, aside from my silly mistake.  
Hopefully this can help others in the future though!
